std/container/slist.d

```diff
@@ -285,6 +285,27 @@ Complexity: $(BIGOH 1)
     }
 
 /**
+Reverses SList in-place. Performs no memory allocation.
+
+Complexity: $(BIGOH n)
+     */
+    void reverse()
+    {
+        if (!empty)
+        {
+            Node* prev;
```

Comment:
I thought of it, but decided against as everyone who uses D knows about default initialization. Also explicit initialization to default value never used in other algorithms of `SList`.